airflow/security/kerberos.py

```diff
@@ -27,19 +27,21 @@
 log = LoggingMixin().log
 
 
-def renew_from_kt():
+def renew_from_kt(args=None):
```

Comment:
I think rather than pass the whole args dict in here it would be better to pass the two we specifically want (`principal`, and `keytab`)

Reply:
okay, make sense